common/app/views/fragments/amp/storyPackageAmp.scala.html

```diff
@@ -19,9 +19,11 @@
 
 @onward(content: CuratedContent) = {
     <div class="fc-item @TrailCssClasses.toneClass(content)">
-        <div class="fc-item__image-container">
-            <amp-img src="@content.properties.maybeContent.flatMap(_.trail.thumbnailPath)" layout="fixed" width=126 height=75></amp-img>
-        </div>
+        @if(content.properties.maybeContent.flatMap(_.trail.thumbnailPath)) {
```

Comment:
shall we put `content.properties.maybeContent.flatMap(_.trail.thumbnailPath)` in a variable?